velox/exec/Operator.h

```diff
@@ -301,69 +301,75 @@ class Operator : public BaseRuntimeStatWriter {
 
   virtual ~Operator() = default;
 
-  // Returns true if 'this' can accept input. Not used if operator is a source
-  // operator, e.g. the first operator in the pipeline.
+  virtual void init();
```

Comment:
Let's add documentation to explain that this method is for doing any construction work that requires memory allocations. Let's also comment that Operator's constructor should not allocate memory.

Reply:
Updated. Thanks!